marklogic-data-hub-central/ui/src/components/flows/new-flow-dialog/new-flow-dialog.tsx

```diff
@@ -25,7 +25,7 @@ interface Props {
 const NewFlowDialog: React.FC<Props> = (props) => {
 
   const [flowName, setFlowName] = useState("");
-  const [description, setDescription] = useState(props.flowData && props.flowData !== {} ? props.flowData.description : "");
```

Comment:
@littlecherro Are we adding this condition somewhere ?

Reply:
@Sanjeevani19  No, that condition doesn't work because `props.flowData` is ALWAYS different to `{}` and there's a useEffect that already loads `flowData.description` to `description` below. Also, this was what generates the "red" warning in the first place.